package-lock.json

```diff
@@ -1,222 +1,8 @@
 {
```

Comment:
Why is this change here?

Reply:
I dont remember altering this, let me look into it.

Reply:
Can you please look into it

Reply:
Ok

Reply:
i am having hard time identifying the root cause here, all i did was run the npm install, and it rewrites the package-lock.json, please tell if you have any idea as to why it is behaving like this @TanishqSingla

Reply:
one more thing do we even need the packages object? 
It is mentioning the same things that we have inside the dependencies object, then why do we even need packages object?
@TanishqSingla

Reply:
done @TanishqSingla